index.ts

```diff
@@ -138,14 +175,17 @@ export function search ({
       query['boundary.rect.max_lon'] = boundary.rect.maxLon
     }
     if (boundary.circle) {
-      const {lat, lon} = lonlat(boundary.circle.centerPoint)
+      // TODO: not sure what to do here... It's clearly wrong
```

Comment:
I'm not sure what this comment means. We should either fix (if it's a big concern) or add more context to the comment.

Reply:
I may have confused myself, but I think https://github.com/conveyal/lonlat/blob/948f87a1da22f3e4962841bd0c6b3e2e75e9a2d9/index.ts#L191 won't handle this single `centerPoint` correctly. But if the library has worked like this for years then maybe it doesn't make sense to change it? I've added a comment in 165129c2154d6b79a50f8b25ee3295a757fccfee, but I might remove it if I've made a mistake here...

Reply:
Removed in c8987d80f93b051d137e728881e9eb94da1aab5b